server/controller/index.js

```diff
@@ -5,11 +5,12 @@ const express = require('express');
 const { checkAuth } = require('./middleware/authentication');
 const { protectRoutes } = require('./middleware/protectRoute');
 const adminHandler = require('./accountsMangment/admin');
+const { getCustomer } = require('./accountsMangment/customer/getCustomers');
 const customerHandler = require('./accountsMangment/customer');
```

Comment:
there is customer folder for put all routes for the customer inside it and you need just add the index of this folder as wrote above 'customerHandler'

Reply:
is it true to be a separated file to handle routes for each part??

Reply:
in our project is required to make it